### PR TITLE
[Docs] Add an example how to use `log_artifact` method 

### PR DESCRIPTION
**Pull request by GiladShapira94**

### CODE REVIEW

**Review by jillnogold — APPROVED** [review]

**Review by jillnogold — APPROVED** [review]

File: docs/store/log-artifacts.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+(log-artifacts)=
+# Logging Artifacts
+To log artifacts that are not of kind model or dataset, use the {py:meth}`~mlrun.execution.MLClientCtx.log_artifact` method.
+You can apply this nethod to the project object or the context object when logging an artifact in runtime, for example a job.
```
Comment (alonmr) [inline docs/store/log-artifacts.md]:
```suggestion
You can apply this method to the project object or the context object when logging an artifact in runtime, for example a job.
```

File: docs/store/log-artifacts.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+# Update layout
+fig.update_layout(
+    title="Sin(x) Plot",
+    xaxis_title="x",
+    yaxis_title="sin(x)",
+    template="plotly_dark" 
+)
+project.log_artifact("plotly-art",
+                     body=fig.to_html(),# convert object for logging an html file
+                     format="html")
```
Comment (alonmr) [inline docs/store/log-artifacts.md]:
```suggestion
project.log_artifact("plotly-artifact",
                     body=fig.to_html(), # Convert object for logging an html file
                     format="html")
```

File: docs/store/log-artifacts.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+```
+## Logging Plotly artifacts 
+This example illustrates using MLRun to convert and handle the object: 
+```python
+# use mlrun to convert the python object
+plotly_artifact = mlrun.artifacts.PlotlyArtifact(figure=fig, key="sin_x") 
+# log the artifact
+context.log_artifact(plotly_artifact)
```
Comment (alonmr) [inline docs/store/log-artifacts.md]: To be consistent with the other comments
```suggestion
# Use mlrun to convert the python object
plotly_artifact = mlrun.artifacts.PlotlyArtifact(figure=fig, key="sin_x") 
# Log the artifact
context.log_artifact(plotly_artifact) 
```

**Review by alonmr — CHANGES_REQUESTED** [review]
Minor typos and conventions

**Review by jillnogold — APPROVED** [review]

File: docs/store/log-artifacts.md [hunk 4]
```diff
@@ -19,10 +19,23 @@ You can apply this method to the project object or the context object when loggi
 Each object type requires a different way to convert it for logging the object to a file. This is just an example of one type.
 ```
 To log an artifacts file, specify the local file path to the file using the `local_path`, or use the `body` to dump the object content.
+
```
Comment (jillnogold) [inline docs/store/log-artifacts.md:22]: **Log with local path**
```python
    with open("file.txt","w") as f:
        f.write("abc is 123")

    project.log_artifact(
        "file-example",
        local_path="file.txt",
        labels={"Test": "label-test"},
    )
```

**Log with body**
```python
    project.log_artifact(
        "some-data",
        body=b"abc is 123",
        format="txt",
        labels={"Test": "label-test"},
    )
```

**Review by jillnogold — APPROVED** [review]

**Review by alonmr — APPROVED** [review]

**Pull request merged**